Gopkg.toml

```diff
@@ -80,9 +80,13 @@ required = [
   name = "github.com/giantswarm/errors"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```

Comment:
This is to be reverted once https://github.com/giantswarm/helmclient/pull/119 gets merged.